src/scenarios/UK.ts

```diff
@@ -0,0 +1,47 @@
+import { Scenario } from '@src/simulator/SimulatorModel';
+
+export const GDP_Uk = 2.8e13; // 2.8 trillion UK gdp source: https://www.google.com/search?sxsrf=ALeKk01S5_HLcnpJFgfGjmYjN2t1XLmflw%3A1612101247616&ei=f7YWYPudJfGEhbIPyM-I4AE&q=uk+gdp&oq=uk+gdp&gs_lcp=CgZwc3ktYWIQAzIECCMQJzIICAAQsQMQkQIyBAgAEEMyAggAMgQIABBDMgQIABBDMgUIABCxAzIECAAQQzICCAAyAggAOgcIABBHELADOgQIABBHUIkHWNkHYMUIaAFwAngAgAFwiAFwkgEDMC4xmAEAoAEBqgEHZ3dzLXdpesgBCMABAQ&sclient=psy-ab&ved=0ahUKEwi72dKRqcbuAhVxQkEAHcgnAhwQ4dUDCA0&uact=5
+
+export const UK: Scenario = {
+    totalPopulation: 66650000, // 66 million people UK population source: https://www.google.com/search?sxsrf=ALeKk01y7DAKcHDVprTj0dCWJ3VpMfO7_Q%3A1612101308264&ei=vLYWYKLZD6CDhbIP38q82Ag&q=uk+population&oq=uk+population&gs_lcp=CgZwc3ktYWIQAzIFCAAQsQMyCAgAELEDEIMBMgUIABCRAjICCAAyCAgAELEDEIMBMgYIABAHEB4yAggAMgIIADICCAAyAggAOgQIABBHOggIABAHEAoQHjoICAAQCBAHEB46BAgjECc6CwgAELEDEIMBEJECOggIABCxAxCRAjoHCAAQsQMQQ1COFVjjHGC-HWgAcAN4AIABU4gBhwSSAQE3mAEAoAEBqgEHZ3dzLXdpesgBCMABAQ&sclient=psy-ab&ved=0ahUKEwiip8iuqcbuAhWgQUEAHV8lD4sQ4dUDCA0&uact=5
+    initialNumDead: 107000, // 107k deaths so far source: https://github.com/CSSEGISandData/COVID-19
+    initialNumInfected: 53285, // New daily cases Jan 1st for UK, source: https://github.com/CSSEGISandData/COVID-19
+    initialMedicalCosts: 1252215000,
+    initialEconomicCosts: 10576809135,
+    initialDeathCosts: 19173932258,
+    r0: 3.42, 
```

Comment:
R is now R0. Containment policies modify R now

For example the starting lockdown policy reduces r by ~70% down to ~1.05. 
Another example: Opening schools only reduces r by ~50% so the r is higher, around ~1.2.